ts/IterableInbox.tsx

```diff
@@ -165,8 +165,13 @@ const IterableInbox = ({
       fetchInboxMessages()
    }
 
-   function returnToInbox() {
-      reset()
+   function returnToInbox(callback: Function) {
```

Comment:
Is there a way to let this parameter be optional, so that if nothing is entered, the function doesn't do any callback? It would make [this call](https://github.com/Iterable/react-native-sdk/pull/294/files#diff-f4c0dd9039a860fa23d99f3faa1f7bbd72be7240d8dcc0117824979ec837b051R117) more intuitive if it was just `returnToInbox()`, instead of needing a blank callback.